models/classes/runner/QtiRunnerService.php

```diff
@@ -788,19 +744,11 @@ public function getToolsStates(RunnerServiceContext $context)
      * @param $itemRef
      * @param $response
      * @return mixed
-     * @throws \common_Exception
+     * @throws common_Exception
      */
     public function parsesItemResponse(RunnerServiceContext $context, $itemRef, $response)
     {
-        if (!$context instanceof QtiRunnerServiceContext) {
-            throw new InvalidArgumentTypeException(
-                'QtiRunnerService',
-                'storeItemResponse',
-                0,
-                QtiRunnerServiceContext::class,
-                $context
-            );
-        }
+        $this->assertIsQtiRunnerServiceContext($context, 'parsesItemResponse');
```

Comment:
Are we sure it is `parsesItemResponse`? Before it was different https://github.com/oat-sa/extension-tao-testqti/pull/2383/files#diff-71b5efb930e0d9c89254a92e58d13ee44bbac13c24942355bab23142687c1b55L798